extensions/blocks/podcast-player/edit.js

```diff
@@ -56,7 +101,7 @@ const PodcastPlayerEdit = ( {
 	 *
 	 * @param {object} event Form on submit event object.
 	 */
-	const checkPodcastLink = event => {
+	const checkPodcastLink = useCallback( event => {
```

Comment:
Minor: we [probably don't need `useCallback`](https://kentcdodds.com/blog/usememo-and-usecallback) here. As with `useMemo` the primary use cases for this are:

There are specific reasons both of these hooks are built-into React:

1. Referential equality
2. Computationally expensive calculations

I don't think we have either of these here so if anything it's probably adding extra overhead to call the `useCallback` hook.

Reply:
Yeah, good point. I've seen that we use callback also when the bound function is tied to an event, like as in this case, `onSubmit`. Let's keep this comment as `unresolved` for now.